ce/session.ts

```diff
@@ -203,8 +201,8 @@ export class Session {
 
   }
 
-  get telemetryEnabled() {
-    return !!this.configuration.globalSettings.get('send-anonymous-telemetry');
+  get telemetryEnabled(): Promise<boolean> {
+    return this.fileSystem.exists(this.homeFolder.join('vcpkg.disable-metrics'));
```

Comment:
the *presence* of the file disables it right? (==`false`) 
Probably should be 

`return !this.homeFolder.exists('vcpkg.disable-metrics');`

Reply:
Megaderp on my part, you're right the sense was flipped. Unfortunately since exists returns a promise a straightforward ! doesn't do the right thing here. I know we normally like to use async/await but I'm having difficulty getting that to work with a getter, so I used the `.then` form for this negation given how simple it is.